Validation/SiTrackerPhase2V/python/Phase2TrackerValidateDigi_cfi.py

```diff
@@ -24,7 +24,8 @@
     SimTrackSource = cms.InputTag("g4SimHits"),
     SimVertexSource = cms.InputTag("g4SimHits"),
     GeometryType = cms.string('idealForDigi'),
-    PtCutOff      = cms.double(9.5),                           
+    #PtCutOff      = cms.double(9.5),
+    PtCutOff      = cms.double(2.0),
```

Comment:
what is the rationale behind this change of the pT cut?

Reply:
Just wanted to check the missedDigiEta, Pt distributions. For tracks with pT ~ 10 GeV, there were no missed Digis. So, I made that pT cutoff lower to see whether there is any Digi not matched with any simulated tracks.

Reply:
@emiglior do you have any recommendation about the pT cutoff for the matching?